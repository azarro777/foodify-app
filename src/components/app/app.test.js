import React from 'react';
import App from './app';
import { shallow } from 'enzyme';

describe('Testing <App/> component', () => {
  const app = shallow(<App />);
  it('App have rendered correctly', () => {
    expect(app).toMatchSnapshot();
  });
});
