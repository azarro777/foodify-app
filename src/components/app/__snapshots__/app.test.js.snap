// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Testing <App/> component App have rendered correctly 1`] = `ShallowWrapper {}`;
